add rough crew assigning

conalli opened the pull request:
very basic but it is a start

khatakslied commented on scripts/scene/EditingManager.gd:
```diff
@@ -98,6 +93,10 @@ func _on_editing_state_input(event: InputEvent) -> void:
+		if event.pressed:
+			match event.button_index:
+				1: room_editor.handle_select_input(event, camera.offset, camera.zoom);
 	
```
_on_editing_state_input does not seem to trigger now. Shall we implement the same 'tooltip' for the room with basic info (capacity, resources etc). You mentioned adding areas to the rooms. Is this what you planned to do, @conalli ?

conalli replied:
It was not possible to do both the assigning and editing so I just temporarily stopped the room edit for now, both room edit and assigning will need to be updated so that they can work together but we need to decide how to do that

khatakslied replied:
Understood. Sorry, if you told me that before and I forgot.

khatakslied commented on scripts/scene/EditingManager.gd:
```diff
@@ -33,11 +33,6 @@ func _ready() -> void:
-		if event.pressed:
-			match event.button_index:
-				1: room_editor.handle_select_input(event, camera.offset, camera.zoom);
 
```
According to @jazmand, the _input is always listening for click, but the load_room_types is just called once, right?

conalli replied:
yeah the load_room_types is just called once on _init, it is not related to the _input function

conalli commented:
@khatakslied @jazmand

added a few small changes just so everything works as before

- removed unnecessary handle_select_input mentioned in ryans comment
- listen for input only on default state for editor so as not to clash with assigning
- rebake the navigation mesh after deleting
